extensions/standard-processors/tests/unit/GetFileTests.cpp

```diff
@@ -292,3 +292,26 @@ TEST_CASE("GetFile sets attributes correctly") {
     CHECK(flow_file->getAttribute(minifi::core::SpecialFlowAttribute::FILENAME) == "beta.txt");
   }
 }
+
+TEST_CASE("GetFile can use expression language in Directory property") {
+  using minifi::processors::GetFile;
+  LogTestController::getInstance().setTrace<GetFile>();
+
+  const auto get_file = std::make_shared<GetFile>("GetFile");
+  minifi::test::SingleProcessorTestController test_controller(get_file);
+
+  std::filesystem::path base_dir = test_controller.createTempDirectory();
+  auto date_str = date::format("%Y-%m-%d", std::chrono::time_point_cast<std::chrono::seconds>(std::chrono::system_clock::now()));
```

Comment:
nitpicking, but why do we cast (from microseconds?) to seconds if we only use the date part?

Reply:
Good point, it's not needed here, updated in a9ebc13605cf3ac4ba35fd6816ee224f3395a9a9. The reason for this was that I started working with getDateTimeStr which takes `std::chrono::sys_seconds`, but after removing it's not needed anymore.